packages/x-data-grid/src/components/containers/GridRootStyles.ts

```diff
@@ -41,6 +41,15 @@ const columnHeaderStyles = {
   },
 };
 
+declare module '@mui/material' {
+  interface Mixins {
+    DataGrid: {
```

Comment:
Currently [used](https://github.com/mui/mui-x/blob/3d86411074f2fe291d42edfa4b930f9baa951f62/packages/x-data-grid/src/themeAugmentation/props.ts#L9) keyword is **MuiDataGrid**, maybe align it, or rename it everywhere? (it might be a BC though 🤔)